pyk/src/pyk/kast/att.py

```diff
@@ -46,7 +49,11 @@ def to_dict(self, value: None) -> Any:
         assert value is None
         return ''
 
-    def pretty(self, value: None) -> None:
+    def unparse(self, value: None) -> None:
+        return None
+
+    def parse(self, text: str) -> None:
+        assert text == ''
```

Comment:
This is not something that we can address right now, just something we eventually should:
* #4027